scss/navbar.scss

```diff
@@ -2,30 +2,25 @@ $nav-height: 90px;
 $nav-logo-height: 30px;
 
 .navbar {
+  align-items: center;
   background-color: $white;
-  z-index: 1;
+  box-shadow: 0 2px 8px $grey;
+  display: flex;
   height: $nav-height;
+  justify-content: space-between;
   position: fixed;
   top: 0;
   width: 100%;
-  box-shadow: 0px 2px 8px $grey;
+  z-index: 1;
 
-  &::before {
-    background: url('assets/wolox_logo.svg') no-repeat;
-    content: '';
+  .navbar-logo {
     height: $nav-logo-height;
-    position: absolute;
-    top: 30px;
-    left: 35px;
-    width: 100%;
+    padding-left: 30px; 
```

Comment:
this should be as `padding: 0 30px;` in the parent. padding is used to separate the parent from its children